sqlite3.go

```diff
@@ -1893,6 +1893,34 @@ func (c *SQLiteConn) SetFileControlInt(dbName string, op int, arg int) error {
 	return nil
 }
 
+// SetFileControlInt invokes the xFileControl method on a given database. The
+// dbName is the name of the database. It will default to "main" if left blank.
+// The op is one of the opcodes prefixed by "SQLITE_FCNTL_". The arg argument
+// and return code are both opcode-specific. Please see the SQLite documentation.
+//
+// This method is not thread-safe as the returned error code can be changed by
+// another call if invoked concurrently.
+//
+// Prefer this method over SetFileControlInt when the argument to the underlying
```

Comment:
I think this method should say "only use if the argument is a pointer to a sqlite3_int64". Meanwhile `SetFileControlInt` should say "useSetFileControlInt64 instead if the argument is a pointer to a sqlite3_int64".

Reply:
Good call out. I've updated the documentation in https://github.com/mattn/go-sqlite3/pull/1298/commits/9882f8f441bfda9333ad5f70119d5692ff960694